app/src/components/DataRequirements.tsx

```diff
@@ -0,0 +1,159 @@
+  const codeFilters = props.codeFilter;
+  const codes: codeInterface[] = [];
+
+  const valueSets = props.codeFilter.filter(function (x: any) {
+    return x?.valueSet !== undefined;
+  });
+
+  codeFilters?.forEach((arr: { code: { code: any; system: any }[] }) => {
+    arr?.code?.forEach((element: { code: any; system: any }) => {
+      const code: codeInterface = { code: element?.code, system: element?.system };
+      codes.push(code);
+    });
+  });
+
+  return (
+    <Center>
+      <Paper className={classes.card} shadow="sm" p="md">
+        <Accordion variant="contained" radius="lg" defaultValue="customization">
+          <Accordion.Item value="customization">
+            <Accordion.Control>
+              {' '}
+              <Center>
+                <h2>{type} </h2>
+              </Center>
+            </Accordion.Control>
+            <Accordion.Panel>
+              <List>
+                <List.Item>
+                  <b>Type: </b>
+                  {type}
+                </List.Item>
+                <br />
+                <List.Item>
+                  <b>Extension: </b>
+                  <br />
+                </List.Item>
+                <List withPadding>
+                  {extensions.map((item: any, index: any, index2: any) => (
+                    <>
+                      {item?.url != undefined && (
+                        <List.Item key={index}>
+                          <b> URL: </b>
+                          {item.url}{' '}
+                        </List.Item>
+                      )}
+                      {item?.valueString != undefined && (
+                        <List.Item key={index2}>
+                          <b> ValueString: </b>
+                          {item.valueString}{' '}
+                        </List.Item>
+                      )}
+                      <br />
+                    </>
+                  ))}
+                </List>
+                {dateFilters?.length > 0 && (
+                  <>
+                    <List.Item>
+                      <b>DateFilter:</b>
+                    </List.Item>
+                    <List withPadding>
+                      {dateFilters?.map((item: any, index: any, index2: any) => (
+                        <>
+                          <List.Item key={index}>
+                            <b> Start Date: </b>
+                            {new Date(item.valuePeriod.start).getUTCMonth() + 1}/
+                            {new Date(item.valuePeriod.start).getUTCDate()}/
+                            {new Date(item.valuePeriod.start).getUTCFullYear()} <br />
+                          </List.Item>
+                          <List.Item key={index2}>
+                            <b> End Date: </b> {new Date(item.valuePeriod.end).getUTCMonth() + 1}/
+                            {new Date(item.valuePeriod.end).getUTCDate()}/
+                            {new Date(item.valuePeriod.end).getUTCFullYear()} <br />
```

Comment:
I like the look of displaying the period start/end with this format, but since the period start/end can be a dateTime, it might be best to just display `item.valuePeriod.start` and `item.valuePeriod.end` as is, without any sort of conversion. That way, we will still have the millisecond precision. This isn't super important for the sake of this demo application, but it might be nice to have this precision so that our precision is consistent across all of our applications (for example, the fqm-execution library uses millisecond precision for date periods).

Reply:
I agree it would be more precise, but I've noticed when I don't convert the time objects, it only ever shows 0's for the milliseconds. This screenshot is an example of what they all look like: 
<img width="496" alt="Screenshot 2023-06-29 at 12 29 08 PM" src="https://github.com/projecttacoma/measure-repository/assets/98363677/e87e6402-bebd-4128-90ed-cff94e1c36c3">

Are they all supposed to have 0's or is it not working as intended?

Reply:
If you compare the start date and end date that are shown here to the `valuePeriod` start and end on the raw data requirements, they should match up. My guess is that for this data requirement, yes the precision should be all 0's because that is what is listed on the data requirement.

Example:
Readable data requirements:
<img width="368" alt="Screenshot 2023-06-30 at 11 13 11 AM" src="https://github.com/projecttacoma/measure-repository/assets/87094479/19326b3b-631e-46ac-b7d0-00c8f0d246a5">

Corresponding value period:
<img width="378" alt="Screenshot 2023-06-30 at 11 13 32 AM" src="https://github.com/projecttacoma/measure-repository/assets/87094479/e5333887-46ca-45bb-9050-e22b7322a3e7">

(above example is for the first Encounter data requirement for the ColorectalCancerScreeningsFHIR measure)